pkgs/tools/networking/webalizer/default.nix

```
@@ -2,24 +2,26 @@
 
 stdenv.mkDerivation rec {
   pname = "webalizer";
-  version = "2.23-05";
+  version = "2.23.08";
 
   src = fetchurl {
-    url = "ftp://ftp.mrunix.net/pub/webalizer/webalizer-${version}-src.tar.bz2";
-    sha256 = "0nl88y57a7gawfragj3viiigfkh5sgivfb4n0k89wzcjw278pj5g";
+    url = "http://ftp.debian.org/debian/pool/main/w/webalizer/webalizer_${version}.orig.tar.gz";
+    sha256 = "sha256-7a3bWqQcxKCBoVAOP6lmFdS0G8Eghrzt+ZOAGM557Y0=";
   };
 
   # Workaround build failure on -fno-common toolchains:
   #   ld: dns_resolv.o:(.bss+0x20): multiple definition of `system_info'; webalizer.o:(.bss+0x76e0): first defined here
   env.NIX_CFLAGS_COMPILE = "-fcommon";
 
+  installFlags = [ "MANDIR=\${out}" ];
```

```suggestion
  installFlags = [ "MANDIR=\${out}/share/man/man1" ];
```

Without that change, the man files would just end up in the root of the built derivation and thus not in paths that are searched by `man`. The hardcoded man1 is needed since the installer doesn't create the category folders itself and it only contains *.1 man files.